esphome/components/lcd_base/lcd_display.cpp

```diff
@@ -148,6 +148,11 @@ void LCDDisplay::printf(const char *format, ...) {
   if (ret > 0)
     this->print(0, 0, buffer);
 }
+void LCDDisplay::clear(void){
```

Comment:
```suggestion
void LCDDisplay::clear() {
```

See https://travis-ci.org/esphome/esphome/jobs/600625094#L910